tests/test_misc.py

```diff
@@ -330,29 +330,69 @@ def test_dict_to_csv_column_list(mock_output_manager: OutputManager) -> None:
     assert v.to_list() == data['values']
 
     result = mock_output_manager._dict_to_csv_column_list(data, ["values", "info_maps"])
+    assert len(result) == 3
     (_, data_series) = result[0]
-    (_, info_maps_series) = result[1]
-    assert data_series.to_list() == data['values'] and info_maps_series.to_list() == data['info_maps']
+    (_, map1_series) = result[1]
+    (_, map2_series) = result[2]
+    assert data_series.to_list() == data['values'] \
+        and map1_series.to_list() == ['value1', 'value2'] \
+        and map2_series.to_list() == [1, 2]
```

Comment:
I think having separate assert statements would help with potential debugging vs. one longer one.
```suggestion
    assert data_series.to_list() == data['values'] 
    assert map1_series.to_list() == ['value1', 'value2']
    assert map2_series.to_list() == [1, 2]
```